golang/main.go

```diff
@@ -21,6 +22,9 @@ func check(e error) {
 	}
 }
 
+var accessToken string
+var err error
```

Comment:
not a good idea to have the global error, we can have global `accessToken`, but create error every time:
```golang
accessToken, err := getAccessToken(token)
```